### PR TITLE
Comma seperated atom values

### LINKED ISSUE #2701: Comma seperate atom values
Decred values are comma separated, but atom values are not. This is problematic because atom values are usually longer and thus need comma separation even more than Decred values.

Screenshot from 2020-10-07 12-51-32

### PR DESCRIPTION
**Pull request by bgptr**
closes #2701

### CODE REVIEW

File: app/components/shared/Balance.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const Balance = ({
     : "balance-small";
   if (currencyDisplay === DCR) {
     let totalDcr = 0;
-    if (typeof amount !== "undefined" && amount !== 0 && !isNaN(amount)) {
+    if (typeof amount !== "undefined" && amount !== 0 && !isNaN(amount) && amount !== null) {
```
Comment (vctt94) [inline app/components/shared/Balance.js:31]: This can be changed to: `amount && !isNan(amount)`

File: app/components/shared/Balance.js [hunk 2]
```diff
@@ -66,10 +66,14 @@ export const Balance = ({
     <div className={classNameWrapper}>
       <span className="mono" {...{ onClick }}>
         <span className={[secondary, bold ? "bold" : null].join(" ")}>
-          {amount + " "}
+          <FormattedNumber
+            value={typeof amount === "undefined" || isNaN(amount) ? 0 : amount}
```
Comment (vctt94) [inline app/components/shared/Balance.js:70]: maybe this can also be simplified?

**Review by vctt94 — COMMENTED** [review]
Good job on this PR, I like you added test for it. 

I left just a couple comments.

**Comment by bgptr** [comment]
Thanks for your review, I like your suggestion. I simplified the conditions.

**Review by alexlyp — APPROVED** [review]

**Pull request merged**